src/features/locators/components/LocatorEditDialog.tsx

```diff
@@ -215,6 +223,20 @@ export const LocatorEditDialog: React.FC<Props> = ({
     updateRef(OnbrdStep.EditLocator, undefined, isOkButtonDisabled ? undefined : handleCreateCustomLocator);
   };
 
+  const renderValidationMessage = () => {
+    return validationMessage === LocatorValidationErrors.DuplicatedLocator ? (
+      <MessageForDuplicate
```

Comment:
please, pay attention that all component names of locator's feature start with `Locator`

Reply:
fixed